client/webserver/site/src/js/markets.ts

```diff
@@ -881,6 +884,20 @@ export default class MarketsPage extends BasePage {
     page.candleVol.textContent = Doc.formatCoinValue(candle.matchVolume, this.market.baseUnitInfo)
   }
 
+  showPreOrderAdvancedOptions () {
+    this.preOrder(this.parseOrder(), true)
```

Comment:
@buck54321 I'm just checking out this PR for the first time now since https://github.com/decred/dcrdex/issues/1874 was your issue, but does it seem right to you that toggling the advanced options should trigger a `fetchPreorder` (esp. if hiding)?  I'm not seeing why the "advanced" options cannot just be un-hidden.

Reply:
No, we shouldn't need to do this just when showing/hiding.